test/modules/post/test/file.rb

```diff
@@ -32,8 +32,8 @@ def initialize(info={})
   #
   def setup
     @old_pwd = pwd
-    tmp = (directory?("/tmp")) ? "/tmp" : "%TMP%"
-    vprint_status("Setup: changing working directory to #{tmp}")
+    tmp = (directory?("/tmp")) ? "/tmp" : "%TEMP%"
+    puts "Setup: changing working directory to #{tmp}"
```

Comment:
That was unintentional to change to puts. I just didn't see vprint_status showing up anywhere, so had meant to switch it temporarily while debugging. What is the proper way to get vprint_status to show up in a place I can monitor?